test/libsolidity/semanticTests/constants/constants_at_file_level_referencing.sol

```diff
@@ -0,0 +1,28 @@
+uint256 constant c = a.length + 56;
+bytes constant a = b;
+bytes constant b = hex"030102";
+
+contract C {
+    function f() public returns (bytes memory) {
+        return a;
+    }
+
+    function g() public returns (bytes memory) {
+        return b;
+    }
+
+    function h() public returns (bytes memory) {
+        return bytes(c);
+    }
+
+    function i() public returns (uint, ActionChoices, bytes32) {
+        return (x, choices, st);
```

Comment:
where are these declared?

Reply:
ah the other test

Reply:
This file already creates a segfault at an earlier time in the processing, but the other test works fine.